views/includes/page-templates/beneficial-owner-gov.html

```diff
@@ -0,0 +1,56 @@
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% include "includes/list/errors.html" %}
+
+      <span class="govuk-caption-xl">Beneficial owner</span>
+      <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
+      <p class="govuk-caption-l  govuk-!-margin-bottom-5 govuk-!-margin-top-5">You can add more later.</p>
+
+      <form class="form" method="post">
+
+        {% include "includes/inputs/fields/name-input.html" %}
+
+        {% include "includes/inputs/address/principal-address-input.html" %}
+
+        {% set hintText = "For example, limited company or partnership" %}
+        {% include "includes/inputs/legal-form-input.html" %}
+
+        {% set hintText = "This is the law that the beneficial owner operates under" %}
+        {% include "includes/inputs/governing-law-input.html" %}
+
+        {% set dateText = "When did it become a beneficial owner for the overseas entity?" %}
+        {% include "includes/inputs/date/start-date-input.html" %}
+
+        {% call govukFieldset({
+          legend: {
+            text: "What is its nature of control?",
+            classes: "govuk-fieldset__legend--m",
+            isPageHeading: false
+          }
+        }) %}
+
+        {% set errorMessage = errors.beneficial_owner_nature_of_control_types if errors %}
+        {% include "includes/inputs/nature-of-control-checkboxes/beneficial-owner-nature-of-control-types.html" %}
+
+        {% set errorMessage = {} if errors.beneficial_owner_nature_of_control_types %}
+        {% include "includes/inputs/nature-of-control-checkboxes/non-legal-firm-nature-of-control-types.html" %}
+
+        {% endcall %}
+
+        {% set sanctionsListText = "Is it on the sanctions list?" %}
+        {% set sanctionsListHintText = "This means that it is subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
+        {% include "includes/inputs/fields/is-on-sanctions-list-input.html" %}
+
+        {% include "includes/inset-text.html" %}
+
+        {% include "includes/save-and-continue-button.html" %}
+
+        {% set removePathURL = OE_CONFIGS.BENEFICIAL_OWNER_GOV_URL + OE_CONFIGS.REMOVE + "/" + id %}
```

Comment:
removePathURL will have to be removed from here and passed in from views/beneficial-owner-gov or views/update/update-beneficial-owner-gov.html with the correct URL